examples/test_flow_dir/test_flow_dir.c

```diff
@@ -214,9 +298,16 @@ main(int argc, char *argv[]) {
         /* Map the sdn_ft table */
         onvm_flow_dir_nf_init();
 
+        if (populate_flow) {
+                printf("Populating flow table. \n");
+                populate_sample_ipv4();
+        }
+
         onvm_nflib_run(nf_local_ctx);
 
         onvm_nflib_stop(nf_local_ctx);
+        flow_entry = NULL;
+        rte_free(flow_entry);
```

Comment:
These as well I think.

Reply:
This should be resolved. I did this as I was DPDK stated I was accessing invalid memory. This solution was misguided,  should be resolved. Thanks!